README.md

```diff
@@ -62,6 +62,27 @@ loggins.trackQNAEvent(context, userQuery, kbQuestion, kbAnswer, score);
 //You can see a working sample in [https://github.com/Microsoft/BotBuilder-CognitiveServices/tree/master/Node/samples/QnAMakerWithFunctionOverrides](https://github.com/Microsoft/BotBuilder-CognitiveServices/tree/master/Node/samples/QnAMakerWithFunctionOverrides)
 ```
 
+## Logging custom properties from session
+
+```js
+var instrumentation = require('botbuilder-instrumentation');
+
+// Setting up advanced instrumentation
+let logging = new instrumentation.BotFrameworkInstrumentation({ 
+  instrumentationKey: ["main insturmnation key","secondary instumentation key"],
+  sentimentKey: process.env.CG_SENTIMENT_KEY,
+  customFields: {
+    userData: [ "CUSTOM_PROPERTY" ]
```

Comment:
I see now that later on you included part of this documentation. But i'd definitely give more background such as the one I mentioned the previous comment.